### PR TITLE
edited pre-reqs to prerequisites

### PR DESCRIPTION
**Pull request by DanielMabadeje**

### CODE REVIEW

**Review by Laerciogj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ For more info, check out the [GitHub documentation](https://docs.github.com/en/f
 ### VS Code Remote - Containers
 Follow these steps to open this sample in a container using the VS Code Remote - Containers extension:
```
Comment (egamma) [inline README.md:19]: pls update grammar

Comment (DanielMabadeje) [reply README.md:19]: ok sure

**Comment by thiwatx33** [comment]
@DanielMabadeje:main

**Comment by Tariqchandio01** [comment]
Bro please  create to my nam command fb clone

**Comment by bamurtaugh** [comment]
Thanks for the changes @DanielMabadeje! Would you also be able to include the following style guideline to ensure consistency:

- "GitHub Codespaces" (capital "C") when referring to the whole product
- Lowercase "codespace" when referring to a singular "codespace," i.e. `Note: If you already have a Codespace or dev container` should be `Note: If you already have a codespace or dev container`

**Review by uhug2 — CHANGES_REQUESTED** [review]
Great chyk

**Pull request merged**